doc/info-to-run-integration-tests-hermez-node.md

```diff
@@ -0,0 +1,34 @@
+# Run specifc hermez-node on integration-testing
+```
+Requirements: docker, docker-compose, nodejs v12.0+, aws cli 2.0+, git, jq
+Set up as env vars AWS_REGION, AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY with vars 
+from paragraph *To push/pull images (hermez-docker user)*
+from there https://www.notion.so/AWS-info-0f116600256c4370b8d9f2be47f961ba
+
+Run those commands to run test:
+git clone git@github.com:hermeznetwork/integration-testing.git
+cd integration-testing
+npm i
+make build-hermezjs
+npm run pretest
+
+// change commit hermez-node in https://github.com/hermeznetwork/integration-testing/blob/main/Makefile#L8
+
+make build-hermez-node
+DEV_PERIOD=3 MODE=coord npx mocha ./test/**/*.mochatest.js --exit > "log/report.txt" 2>&1
```

Comment:
This commands seems weird to me. As far as I know they use the next commands. Could you try them??
`npm i`
`make dependencies`
`make test-config`
`npx hardhat test ./test/test-synch/transactions/l1-l2txs.buidlertest.js`
Run this commands in a clean repo folder

Reply:
- `npm run pretest` does include the `make dependencies` and `make test-config` commands :)
- depending on the test you want to run, in your case, the test with that integrates with hermezJS the command will be slightly different:
  - test without hermezJS: `npx hardhat test #name test`
  - test with hermezJS: `DEV_PERIOD=3 MODE=coord npx mocha ./test/**/*.mochatest.js --exit`
- just added `> "log/report.txt" 2>&1` to save test output to a file

Reply:
It's not silly to add this. Thank you for the suggestion!